ergo/distributions/conditions.py

```diff
@@ -74,6 +94,26 @@ def describe_fit(self, dist):
         description["p_in_interval"] = float(self.actual_p(dist))
         return description
 
+    def get_normalized(self, true_min: float, true_max: float):
+        true_range = true_max - true_min
+        normalized_min = (
+            ((self.min - true_min) / true_range) if self.min is not None else None
```

Comment:
Could consider putting this in a function since it's used twice. Same in `get_denormalized`

Reply:
meh, the function would have too many params

Reply:
What about something like this:

```py
@dataclass
class Scale:
    scale_min: float
    scale_max: float

    def range(self):
        return self.scale_max - self.scale_min
    
    def normalize(self, value):
        return (value - self.scale_min) / self.range() if value is not None else None
    
    def denormalize(self, value):
        ...


class FooCondition:

    def normalize(self, true_min: float, true_max: float):
        scale = Scale(true_min, true_max)
        normalized_min = scale.normalize(self.min)
        normalized_max = scale.normalize(self.max)
        return self.__class__(self.p, normalized_min, normalized_max, self.weight)
```

Reply:
huh --- this sort of seems like a bunch more code and somewhat more abstraction for a slight gain in clarity, once you've read and understood it all

happy to defer to you on this, though, just let me know

Reply:
I think across all of the conditions it will reduce code duplication enough to be worth it. Currently the line `true_range = true_max - true_min` appears 8 times in this file. (Not sure what I sketched is exactly the right solution, but I think there is a solution like this that is better than the current version.)

Reply:
ah got it, I previously misunderstood this somewhat (didn't understand that it was supposed to be able to apply to ~any condition)

I'll see if something like this can work

Reply:
FYI incorporating this into my `wrong-fitted-p` branch (needed to add normalization for mixtures, so figured I might as well clean everything up)